src/index.html

```diff
@@ -22,6 +22,409 @@
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="review card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="review__count">Reviews: 5</div>
+        </div>
+
+        <div class="price card__price">
+          <div class="price__label">Price:</div>
+
+          <div class="price__value">$2,199</div>
+        </div>
+
+        <a
+          class="card__buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="review card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="review__count">Reviews: 5</div>
+        </div>
+
+        <div class="price card__price">
+          <div class="price__label">Price:</div>
+
+          <div class="price__value">$2,199</div>
+        </div>
+
+        <a
+          class="card__buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="review card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="review__count">Reviews: 5</div>
+        </div>
+
+        <div class="price card__price">
+          <div class="price__label">Price:</div>
+
+          <div class="price__value">$2,199</div>
+        </div>
+
+        <a
+          class="card__buy"
+          href="#"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The product card structure is repeated multiple times with the same content. This repetition can be optimized by using a templating engine or JavaScript to dynamically generate these cards. This will make the code more maintainable and reduce redundancy.